sql/driver/conn.go

```diff
@@ -103,16 +92,13 @@ func (c *conn) send(args Request) (*rows, error) {
 	// Set the session state even if the server returns an application error.
 	// The server is responsible for constructing the correct session state
 	// and sending it back.
-	if c.session != nil {
-		panic("connection has lingering session state")
-	}
 	c.session = resp.Session
 	// Translate into rows
-	r := &rows{}
+	resultRows := &rows{}
```

Comment:
Some of this change doesn't seem like cleanup so much as enforcing your own naming style and standards. While I'm not opposed to this renaming, I worry that this churn isn't actually beneficial.